system_tests/logging_.py

```diff
@@ -50,8 +50,18 @@ def setUp(self):
         self.to_delete = []
 
     def tearDown(self):
+        from gcloud.exceptions import NotFound
         for doomed in self.to_delete:
-            doomed.delete()
+            backoff_intervals = [1, 2, 4, 8]
+            while True:
+                try:
+                    doomed.delete()
+                    break
+                except NotFound:
```

Comment:
But if `doomed` is already deleted before the first `doomed.delete()` then it'll just error out 4 times.

As for #1657, can we hold off on a more conclusive answer before implementing?